docs/user/03-traces.md

```diff
@@ -327,55 +320,51 @@ The Istio module is crucial in distributed tracing because it provides the [ingr
+ The Istio module is configured with an [extension provider](https://istio.io/latest/docs/tasks/observability/telemetry/) called `kyma-traces`. To activate the provider on the global mesh level using the Istio [Telemetry API](https://istio.io/latest/docs/reference/config/telemetry/#Tracing), place a resource to the `istio-system` Namespace. The following snippets help setting up the Istio tracing feature:
 
-<div tabs name="Configure Istio traces" group="configuration-options">
-  <details>
-  <summary label="extension-provider">
-  Set up extension provider
-  </summary>
-  The following example configures all Istio proxies with the `kyma-traces` extension provider, which, by default, reports span data to the trace gateway of the Telemetry module.
+<!-- tabs:start -->
 
-  ```yaml
-  apiVersion: telemetry.istio.io/v1alpha1
-  kind: Telemetry
-  metadata:
-    name: tracing-default
-    namespace: istio-system
-  spec:
-    tracing:
-    - providers:
-      - name: "kyma-traces"
-  ```
+#### **Extension Provider**
 
-  </details>
-  <details>
-  <summary label="sampling-rate">
-  Set sampling rate
-  </summary>
-  By default, the sampling rate is configured to 1 percent. That means that only 1 trace out of 100 traces is reported to the trace gateway, and all others are dropped. Hereby, the sampling decision itself is propagated as part of the [trace context](https://www.w3.org/TR/trace-context/#sampled-flag) so that either all involved components are reporting the span data of a trace, or none. Increasing the sampling rate results in much higher network utilization in the cluster and also increases the amount of data sent to your tracing backend. Usually, a very low percentage of around 5% is used in a production setup to reduce costs and performance impacts.
+The following example configures all Istio proxies with the `kyma-traces` extension provider, which, by default, reports span data to the trace gateway of the Telemetry module.
 
-  To configure an "always-on" sampling, configure a sampling rate of 100%:
+```yaml
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: tracing-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: "kyma-traces"
+```
 
-  ```yaml
-  apiVersion: telemetry.istio.io/v1alpha1
-  kind: Telemetry
-  metadata:
-    name: tracing-default
-    namespace: istio-system
-  spec:
-    tracing:
-    - providers:
-      - name: "kyma-traces"
-        randomSamplingPercentage: 100.00
-  ```
+#### **Sampling Rate**
+
+By default, the sampling rate is configured to 1 %. That means that only 1 trace out of 100 traces is reported to the trace gateway, and all others are dropped. The sampling decision itself is propagated as part of the [trace context](https://www.w3.org/TR/trace-context/#sampled-flag) so that either all involved components are reporting the span data of a trace, or none. 
```

Comment:
```suggestion
By default, the sampling rate is configured to 1%. That means that only 1 trace out of 100 traces is reported to the trace gateway, and all others are dropped. The sampling decision itself is propagated as part of the [trace context](https://www.w3.org/TR/trace-context/#sampled-flag) so that either all involved components are reporting the span data of a trace, or none. 
```
I missed this earlier; no need for the blank space here